autogen.sh

```diff
@@ -4,8 +4,6 @@
 srcdir=`dirname $0`
 test -z "$srcdir" && srcdir=.
 
-PKG_NAME="eos-installer"
```

Comment:
You could also replace `gnome-autogen.sh` with `autoreconf -v -f -i`.

Reply:
(cf. https://wiki.gnome.org/Projects/GnomeCommon/Migration)

Reply:
Look, I was just annoyed about seeing a warning at the start of every clean build. I have no plans to do any build system migration until after deleting all the unused gnome-initial-setup code, at which point one may as well move to Meson. I can drop this patch if the idea of removing one warning without doing a total rewrite of `autogen.sh` is controversial.

Reply:
I think what you’ve done here is fine: I was just providing some further information which might be useful if you were planning to drop the gnome-common dependency. Sorry if that wasn’t clear.